README.md

```diff
@@ -32,77 +32,164 @@ architecture.
+```
+NAME                                                             TYPE                                  DATA   AGE
+brigade-2e9bb93bf149536a951d236772ae8be77a3cef9335c82bf39fc18c   brigade.sh/project                    24     2m
+...other cluster Secrets...
 ```
 
-The Helm chart will be removed before Brigade 1.0.0, so we recommend using
-`brig project create` instead. Note that the projects created with the Helm chart
-remain compatible with Brigade.
+### Creating Your First Brigade Build
 
-### Creating Your First `brigade.js`
+Conceptually, a Brigade Build is a set of instructions (written in JavaScript, in a file called `brigade.js`) that will run upon a triggering of an event. Event can be something like a git push, a Docker push or just a webhook. In this way, you can think of Builds like job(s)/task(s)/pipeline(s).
 
-Creating your first `brigade.js` is as easy as this:
+Let's now create a new file called `brigade.js` with the following content:
 
 ```javascript
-const { events } = require("brigadier");
+const { events, Job } = require("brigadier")
+events.on("exec", () => {
+  var job = new Job("do-nothing", "alpine:3.8")
+  job.tasks = [
+    "echo Hello",
+    "echo World"
+  ]
+
+  job.run()
+})
+```
+
+When the `exec` event is triggered, Brigade will create a Build based on this brigade.js file. This Build will create a single job that will start an image based on `alpine:3.8` which will simply do a couple of echoes. The Kubernetes Pod that will be created to run this Build job will have a name starting with 'do-nothing'. As you can probably guess, a new Kubernetes Secret will also be created to store information about this Build.
+
+Moreover, you can check out [this tutorial](https://azure.github.io/brigade/intro/) for more on creating scripts.
+
+### Running a Build
+
+To create and run a Brigade Build for the brigade.js file we wrote, we will use `brig`.
+
+```bash
+brig run deis/empty-testbed -f brigade.js
+```
+
+This will trigger the `exec` event and show you the detailed output, which will be similar to this:
+
+```
+Event created. Waiting for worker pod named "brigade-worker-01d0y7bcxs6ke0yayrx6nbvm39".
+Build: 01d0y7bcxs6ke0yayrx6nbvm39, Worker: brigade-worker-01d0y7bcxs6ke0yayrx6nbvm39
+prestart: no dependencies file found
+prestart: loading script from /etc/brigade/script
+[brigade] brigade-worker version: 0.19.0
+[brigade:k8s] Creating secret do-nothing-01d0y7bcxs6ke0yayrx6nbvm39
+[brigade:k8s] Creating pod do-nothing-01d0y7bcxs6ke0yayrx6nbvm39
+[brigade:k8s] Timeout set at 900000
+[brigade:k8s] Pod not yet scheduled
+[brigade:k8s] default/do-nothing-01d0y7bcxs6ke0yayrx6nbvm39 phase Pending
+[brigade:k8s] default/do-nothing-01d0y7bcxs6ke0yayrx6nbvm39 phase Succeeded
+done
+```
+
+As you can see, Brigade created a new pod for this Build (called `do-nothing-01d0y7bcxs6ke0yayrx6nbvm39`) that executed our job. Let's get its logs.
+
+```bash
+kubectl logs do-nothing-01d0y7bcxs6ke0yayrx6nbvm39
```

Comment:
Was there a PR at some point that follows the logs directly?
Can we use that?

Reply:
yes, but it's not still in the released version of `brig`. Taking it as a task to update it when we do that.